### PR TITLE
cord19 templates

### PR DESCRIPTION
**Pull request by srulikbd**
hope its good!
if not, send me a messege!

### CODE REVIEW

**Comment by VictorSanh** [comment]
to err on the safe side, let' skip `hate_speech18` as it clearly contains harmful content (extracted from a white supremacist forum...) can you remove it from the pr and mark it as "harmful content" in the spreadsheet?
reviewing the rest

File: templates/cord19/metadata/templates.yaml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+templates:
+  10d78ae0-635d-4cf3-8e24-61c879fd6ae7: !Template
+    id: 10d78ae0-635d-4cf3-8e24-61c879fd6ae7
+    jinja: "Write title for the following abstract: {{abstract}} |||\ntitle: \n{{\
+      \ title }}"
+    name: title generation 1
+    reference: ''
+  1821279d-37a7-42f0-ab0c-2a5589a2a7c3: !Template
+    id: 1821279d-37a7-42f0-ab0c-2a5589a2a7c3
+    jinja: "{{title}}\nThe abstract of this title is:  |||\n {{ abstract }}"
```
Comment (VictorSanh) [inline templates/cord19/metadata/templates.yaml:12]: Replace with:
```jinja
Title: {{title}}
Generate a plausible scientific abstract for a scientific paper on Covid-19 with the previous title
```

File: templates/cord19/metadata/templates.yaml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    id: 21fc3c51-5168-4abb-b969-81a115f2f568
+    jinja: "Write abstract for the following title: {{title}}|||\nabstract: \n{{ abstract\
+      \ }}"
+    name: abstract generation 2
+    reference: ''
+  6a2ebf64-9db7-41f0-85a5-379270c54fa6: !Template
+    id: 6a2ebf64-9db7-41f0-85a5-379270c54fa6
+    jinja: '{{abstract}}.
+
+      The title of this abstract is:  |||
```
Comment (VictorSanh) [inline templates/cord19/metadata/templates.yaml:25]: `What could a scientific title be for this abstract on Covid-19?`

**Review by VictorSanh — CHANGES_REQUESTED** [review]
left comments. see above.

please remove "false" tasks where we know the answer independently of the example.

**Comment by srulikbd** [comment]
I made a new PR based on your comments..thanks!!

**Pull request closed without merging**